src/Concerns/ValidateableData.php

```diff
@@ -52,4 +52,17 @@ public static function withValidator(Validator $validator): void
     {
         return;
     }
+
```

Comment:
Could you rename this to `getValidationRules`?

Reply:
Yeah, will also do that tomorrow with the documentation.